packages/cli/templates/js/field-plugin.config.json

```diff
@@ -0,0 +1,12 @@
+{
+    "options": [
+        {
+            "name": "option1",
```

Comment:
I'm not using the `key` property here as defined in our RFC since we had already a type defined for the `options` schema (used by the Sandbox/Container)

![image](https://github.com/storyblok/field-plugin/assets/1240591/73f447a3-fc3a-4333-b6fd-d9148771c989)

In this type, we have `name` besides `key` and as it's not a big deal, I find it easier to keep the same structure for the options in the manifest options.

![image](https://github.com/storyblok/field-plugin/assets/1240591/a23f3b05-88b8-400f-8d92-7a1f9723bfd8)

This way we don't need to change our types or parse the data. 

What are your thoughts on that?

Reply:
yeah, let's keep it as-is!